frontend/src/scenes/batch_exports/BatchExportScene.tsx

```diff
@@ -370,6 +242,284 @@ export function BatchExportScene(): JSX.Element {
+            color = 'var(--blue)'
+            break
+        case BatchExportLogEntryLevel.Warning:
+            color = 'var(--warning)'
+            break
+        case BatchExportLogEntryLevel.Error:
+            color = 'var(--danger)'
+            break
+        default:
+            break
+    }
+    return <span style={{ color }}>{type}</span>
+}
+
+const columns: ResizableColumnType<BatchExportLogEntry>[] = [
+    {
+        title: 'Timestamp',
+        key: 'timestamp',
+        dataIndex: 'timestamp',
+        span: 3,
+        render: (timestamp: string) => dayjs(timestamp).format('YYYY-MM-DD HH:mm:ss.SSS UTC'),
+    },
+    {
+        title: 'Level',
+        key: 'level',
+        dataIndex: 'level',
+        span: 1,
+        render: BatchExportLogEntryLevelDisplay,
+    } as ResizableColumnType<BatchExportLogEntry>,
+    {
+        title: 'Message',
+        key: 'message',
+        dataIndex: 'message',
+        span: 6,
+    } as ResizableColumnType<BatchExportLogEntry>,
+]
+
+export function LogsTab({ batchExportId }: BatchExportLogsProps): JSX.Element {
+    const { activeTab, batchExportConfig, batchExportConfigLoading } = useValues(batchExportLogic)
+
+    if (!batchExportConfig || batchExportConfigLoading || !activeTab) {
+        return <LemonSkeleton />
+    }
+
+    const logic = batchExportLogsLogic({ batchExportId })
+    const {
+        batchExportLogs,
+        batchExportLogsLoading,
+        batchExportLogsBackground,
+        isThereMoreToLoad,
+        batchExportLogsTypes,
+    } = useValues(logic)
+    const { revealBackground, loadBatchExportLogsMore, setBatchExportLogsTypes, setSearchTerm } = useActions(logic)
+
+    return (
+        <div className="ph-no-capture space-y-2 flex-1">
+            <LemonInput
+                type="search"
+                placeholder="Search for messages containing…"
+                fullWidth
+                onChange={setSearchTerm}
+                allowClear
+            />
+            <div className="flex items-center gap-2">
+                <span>Show logs of type:&nbsp;</span>
+                <Checkbox.Group
+                    options={Object.values(BatchExportLogEntryLevel)}
+                    value={batchExportLogsTypes}
+                    onChange={setBatchExportLogsTypes}
+                    style={{ marginLeft: '8px' }}
+                />
+            </div>
+            <LemonButton
+                onClick={revealBackground}
+                loading={batchExportLogsLoading}
+                type="secondary"
+                fullWidth
+                center
+                disabledReason={!batchExportLogsBackground.length ? "There's nothing to load" : undefined}
+            >
+                {batchExportLogsBackground.length
+                    ? `Load ${pluralize(batchExportLogsBackground.length, 'newer entry', 'newer entries')}`
+                    : 'No new entries'}
+            </LemonButton>
+            <ResizableTable
```

Comment:
Personally I would have considered opting to use our standard LemonTable here instead but for now this is fine

Reply:
Oh, okay, I mostly copied over what I saw in PluginLogs.

Reply:
I'm imagining that if and when we change PluginLogs to LemonTable we can also change this, assuming it would not be adding to much work

Reply:
afaik everything new should use LemonTable

Reply:
I've swapped it out and updated the screenshot 👍